dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -36,7 +36,7 @@ class NumericStatsMixin(with_metaclass(abc.ABCMeta, object)):
     Abstract numerical column profile subclass of BaseColumnProfiler. Represents
     a column in the dataset which is a text column. Has Subclasses itself.
     """
-    col_type = None
+    type = None
```

Comment:
Just curious: does numerical stats even need type?

Reply:
I would need to validate, but I'm not positive it does.